### PR TITLE
change file owner when docker start

### PR DESCRIPTION
**Pull request by dw511214992**

### CODE REVIEW

File: tools/sdk-generation-pipeline/scripts/change-owner.sh [hunk 1]
```diff
@@ -11,14 +11,12 @@ done
 if [ -d "${SPEC_REPO}" ]; then
   while true
```
Comment (zzvswxy) [inline tools/sdk-generation-pipeline/scripts/change-owner.sh:12]: how exit the while loop?

File: tools/sdk-generation-pipeline/scripts/change-owner.sh [hunk 2]
```diff
@@ -11,14 +11,12 @@ done
 if [ -d "${SPEC_REPO}" ]; then
```
Comment (zzvswxy) [inline tools/sdk-generation-pipeline/scripts/change-owner.sh:11]: how to exit the script, for line 7-9 while loop

File: tools/sdk-generation-pipeline/scripts/change-owner.sh (hunk 1 above)
Comment (dw511214992) [reply tools/sdk-generation-pipeline/scripts/change-owner.sh:12]: No need to exit while running. You can check the entrypoint.sh, and you will find it runs in backend.

**Review by zzvswxy — APPROVED** [review]
Checked with Wei:
1 SPEC_REPO will be cloned out of docker
2 docker run this script to change the generated sdk file's owner and group to pipeline's owner and group
3 the script is a resident process, will work until the docker exit

**Pull request merged**